README.md

```diff
@@ -74,21 +76,25 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-
-
-- **[Display my Portfolio]**
+- **[Mobile Version]**
+- **[Using Flexbox]**
+- **[Using Grid]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-<!-- GETTING STARTED -->
+## 🚀 Live Demo <a name="live-demo"></a>
 
-## 💻 Getting Started <a name="getting-started"></a>
+- [Live Demo Link](https://rawcdn.githack.com/peteryeungtt/My-Portfolio/b34888391911b4f667e807021bd5a65a2edfa0e4/index.html)
```

Comment:
- [ ] [Optional] As the project not yet completed and its not in the project requirement to deploy incomplet project please i would suggest you to remove the demo link .

Reply:
@ademibowale I was not suppose to be a full link, but a demo link